### PR TITLE
pinboard-notes-backup 1.0.3 (new formula)

### PR DESCRIPTION
**Pull request by bdesham**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

----

This PR adds a formula for [pinboard-notes-backup](https://github.com/bdesham/pinboard-notes-backup), a command-line utility to back up the notes you’ve saved to [Pinboard](https://pinboard.in).

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by bdesham** [comment]
The build succeeded the second time! 🙂

File: Formula/pinboard-notes-backup.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  head "https://github.com/bdesham/pinboard-notes-backup.git"
+
+  depends_on "ghc@8.0" => :build
+  depends_on "cabal-install" => :build
+
+  def install
+    install_cabal_package
+  end
+
+  # A real test would require hard-coding a Pinboard API key here, which would
```
Comment (ilovezfs) [inline Formula/pinboard-notes-backup.rb]: You can assert an expected failure of some kind instead, or mock the server (see faas-cli)

**Review by ilovezfs — CHANGES_REQUESTED** [review]
The test needs to exercise and verify the functionality of the software.

**Comment by bdesham** [comment]
I don’t think it’s possible to mock the server, since the hostname is hardcoded into the executable. I changed the test to make sure that the program exits with the correct code if the command-line arguments are invalid.

**Comment by bdesham** [comment]
It doesn’t seem to be super clear in GitHub’s interface but I have pushed an updated version (and it has passed the CI checks).

**Review by ilovezfs — APPROVED** [review]

**Comment by fxcoudert** [comment]
Congratulations @bdesham  on your first contribution to Homebrew! We look forward to seeing you again, for example with updates to this formula, or with suggestions on others :)

**Pull request closed without merging**